src/components/ebay-icon/component-browser.js

```diff
@@ -1,28 +1,71 @@
 let rootSvg;
+let svgDefs;
+
+function createLinearGradientFromString(data) {
+    // This is needed because if we add linear gradient as a string it will not be rendered.
+    const linearGradient = document.createElementNS(
+        "http://www.w3.org/2000/svg",
+        "linearGradient"
+    );
+    linearGradient.setAttribute("id", data.id);
```

Comment:
I think you should be able to loop through `data` and get the key/value pairs to use for these `setAtrribute` lines to avoid explicit attributes. That should make it more scalable.

Reply:
See the other comment for an answer to this.

Reply:
If we keep it this way, the next time design introduces a gradient with an attribute previously unused, such as `gradientUnits`, the script will ignore it silently. The resulting icon will not look right, and we'd wonder why. Then, we'd have to troubleshoot to figure out why it doesn't look right and I doubt we'd going to remember this particular nuance in the script or this thread, especially if someone else takes the ticket. Once we find the issue, then we'd have to modify the script to add support for that one attribute and then rerun the script. It seems like if this is changed to more of a passthrough instead of explicitly setting things based on the specific attributes currently used, we avoid all of that.

Reply:
Thats the point. I want to make sure the icon is broken so we can know to modify the script.
I don't know exactly what design will do. When we get there we can update as needed. This will both help double check our code being robust and also implement it then rather than second guessing now.

Reply:
Hmm.. why leave room for the possibility to have the icon be broken when we can create a simple passthrough to avoid all of it ahead of time? This doesn't seem to be over-engineering to me. Ok, I guess I'll go ahead and approve the PR.